test/grid.html

```diff
@@ -150,6 +151,21 @@
       });
     });
 
+    test('Columns per row (#381)', function(done) {
+      container.data = buildDataSet(1000);
+      container.itemSize = 33.3;
+      Polymer.dom.flush();
+      list.scroll(0, 5021);
+
+      requestAnimationFrame(function() {
+        setTimeout(function() {
+          assert.equal(list._physicalItems[list._physicalStart].innerText.trim(), '783');
```

Comment:
This might fix the test in IE 11/10:
```js
 assert.equal(list._physicalStart % list._itemsPerRow, 0);
 assert.equal(list._physicalEnd % list._itemsPerRow, list._itemsPerRow - 1);
```